### PR TITLE
Create 2024-07-26-Sebastes_fasciatus-fSebFas1.md

### PR DESCRIPTION
**Pull request by bhkoo**
Social media post for Sebastes fasciatus (Acadian redfish), fSebFas1 curation

### CODE REVIEW

**Comment by bgruening** [comment]
Thanks @bhkoo! We will take care and propose a template. Thanks a lot!

**Comment by nekrut** [comment]
This should not go to Matrix or Slack. This is for external use, so LinkedIn, Mastodon, and BlueSky only. It should also propagate to central panes om usegalaxy.*

File: posts/2024/2024-07-26-Sebastes_fasciatus-fSebFas1.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+media:
+  - bluesky
+  - mastodon
+  - matrix
+  - linkedin
```
Comment (bgruening) [inline posts/2024/2024-07-26-Sebastes_fasciatus-fSebFas1.md:6]:
```suggestion
  - bluesky-galaxyproject
  - matrix-eu-announce
  - linkedin-galaxyproject
```

File: posts/2024/2024-07-26-Sebastes_fasciatus-fSebFas1.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+mentions:
```
Comment (bgruening) [inline posts/2024/2024-07-26-Sebastes_fasciatus-fSebFas1.md:8]:
```suggestion
mentions:
  bluesky:
    - ebpgenome.bsky.social
```

File: posts/2024/2024-07-26-Sebastes_fasciatus-fSebFas1.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    - usegalaxy
+  mastodon:
+    - VertebrateGenomesProject
+    - usegalaxy
+  matrix:
+    - VertebrateGenomesProject
+    - usegalaxy
+  linkedin:
+    - VertebrateGenomesProject
+    - usegalaxy
```
Comment (bgruening) [inline posts/2024/2024-07-26-Sebastes_fasciatus-fSebFas1.md:22]:
```suggestion
hashtags:
  bluesky:
    - UseGalaxy
  mastodon:
    - VGP
    - EBP
    - UseGalaxy
```

File: posts/2024/2024-07-26-Sebastes_fasciatus-fSebFas1.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+---
+We have finished the curation of Sebastes fasciatus (Acadian redfish), fSebFas1. The files have been uploaded to [GenomeArk](https://genomeark.s3.amazonaws.com/index.html?prefix=species/Sebastes_fasciatus/fSebFas1/assembly_curated/). Curation was done by Kirsty McCaffrey, with QCing by Jo Collins.
+
```
Comment (bgruening) [inline posts/2024/2024-07-26-Sebastes_fasciatus-fSebFas1.md:26]:
```suggestion

![grafik](https://github.com/user-attachments/assets/95eef11f-f445-4b37-a548-d2fdefb7b110)


```

**Comment by bgruening** [comment]
The image could be automatically retrieved from Wikipedia I think. See here: https://grafana.com/blog/2024/07/12/obirdability-how-to-build-an-observability-system-for-bird-songs/?utm_source=grafana_news&utm_medium=rss

File: posts/2024/2024-07-26-Sebastes_fasciatus-fSebFas1.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+Largest gap in scaffolds: 200  
+Smallest gap in scaffolds: 100  
+Base composition (A:C:G:T): 246189749:170749526:170284260:246527338  
+GC content %: 40.90  
+Num soft-masked bases: 0  
+Num segments: 674  
+Total segment length: 833750873  
+Average segment length: 1237019.10  
+Num gaps: 471  
+Num paths: 203
```
Comment (arash77) [inline posts/2024/2024-07-26-Sebastes_fasciatus-fSebFas1.md:62]: We can turn this into an image or link to it to make it look better.

**Pull request closed without merging**